src/actions/actionIndex.js

```diff
@@ -0,0 +1,105 @@
+import axios from 'axios'
+
+const APIkey = 'c905350f371fe191'
+
+export const GET_TODAYS_SUMMARY = 'GET_TODAYS_SUMMARY'
+export const getTodaysSummary = () => {
+    return function(dispatch) {
+        axios.get(`http://api.wunderground.com/api/${APIkey}/forecast/q/CA/San_Francisco.json`)
+        .then(function(res) {
+            dispatch({
+                type: GET_TODAYS_SUMMARY,
+                payload: res.data
+            })
+        })
+        .catch((e) => {console.error(413, 'Error: ', e)})
+    }
+}
+
+
+export const GET_CURRENT_WEATHER = 'GET_CURRENT_WEATHER'
+export const getCurrentWeather = () => {
+	return function(dispatch) {
+        axios.get(`http://api.wunderground.com/api/${APIkey}/hourly/q/CA/San_Francisco.json`)
+      	.then(function(res) {
+            dispatch({
+        		type: GET_CURRENT_WEATHER,
+        		payload: res.data
+        	})
+    	})
+    	.catch((e) => {console.error(412, 'Error: ', e)})
+	}
+}
+
+export const GET_HISTORY_WEATHER = 'GET_HISTORY_WEATHER'
+export const getHistoryWeather = (year) => {
+    return function(dispatch) {
+        const todaysMonth = (new Date()).getMonth() + 1
+        const todaysDate = (new Date()).getDate()
+        const tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000)
+        const tomorrowsMonth = (new Date(tomorrow)).getMonth() + 1
+        const tomorrowsDate = (new Date(tomorrow)).getDate()
+        const getMonthString = (month) => {
+            let formatedMonth = ''
+            if (month < 10) {
+                formatedMonth = '0' + month
+            } else {
+                formatedMonth = month.toString()
+            }
+            return formatedMonth
+        }
+        const getDateString = (date) => {
+            let formatedDate = ''
+            if (date < 10) {
+                formatedDate = '0' + date
+            } else {
+                formatedDate = date.toString()
+            }
+            return formatedDate
+        }
+        //const year = '1957'        
+        const todaysDateInHistory = year + getMonthString(todaysMonth) + getDateString(todaysDate)
+        const tomorrowsDateInHistory = year + getMonthString(tomorrowsMonth) + getDateString(tomorrowsDate)
+        axios.get(`http://api.wunderground.com/api/${APIkey}/history_${todaysDateInHistory}/q/CA/San_Francisco.json`)
```

Comment:
There's a lot of repetition of hard-coded URLs. (e.g. `http://api.wunderground.com/api/`) May be good idea to store the API URL in a `const` e.g. `const API_URL_BASE = 'http://api.wunderground.com/api'`. Also perhaps consider adding/using a `getApiUrl` method. E.g:

```js
const API_KEY = 'osjdflksjflkajsfasdf'
const API_URL_BASE = 'http://api.wunderground.com/api'

const getApiUrl = (date = 2016, state = 'CA', city = 'San_Francisco') =>
  `${API_URL_BASE}/${API_KEY}/history_${date}/q/${state}/${city}.json`

...

axios.get(getApiUrl(1970)).then().etc()

axios.get(getApiUrl(1950)).then().etc()
```